raft/README.md

```diff
@@ -71,6 +71,19 @@ To start a node from scratch:
   n := raft.StartNode(c, []raft.Peer{{ID: 0x02}, {ID: 0x03}})
 ```
 
+To allow a single node cluster, set peers to itself, like so:
```

Comment:
oh. you are right. for the duplication issue, we expect most users start a 3/6 member cluster right away instead of creating a one node cluster and joining other peers later actually.